test/PSRP.Tests.ps1

```diff
@@ -29,23 +29,44 @@ Describe " PowerShell Remoting basic functional tests" -Tag @("CI") {
             Get-PSSession|Remove-PSSession
         }
 
-        It "Remoting from Windows/Linux/MacOS to Linux with basic authentication with winrm/omicli should work" {
+        It "Remoting from Windows/Linux/MacOS to Linux with basic authentication with omicli over https should work" {
```

Comment:
instead of 'Remoting ...' I think I would prefer 'WSMan ...'. That way it is either PowerShell or WSMan, meaning PSRP or CIM operations.